typespec-extension/changelog.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 0.14.0 (2024-02-29)
```

Comment:
Why is this 0.14.0 instead of 0.13.11?

Reply:
The switch to stream-style-serialization is a huge change. (preview https://github.com/Azure/azure-sdk-for-java/pull/38268/files)
And it may take a week to fully move to this (even when no bug found).

Hence, we may had to release patch on 0.13.x.